backend/danswer/connectors/slack/connector.py

```diff
@@ -88,19 +57,38 @@ def _get_channels(
 def get_channels(
     client: WebClient,
     exclude_archived: bool = True,
```

Comment:
changed this function to be able to request public or private channels separately 

added defaults to keep behavior identical